applications/registration-react/src/auth/auth-service.js

```diff
@@ -12,6 +12,12 @@ const PERMISSION_ADMIN = 'PERMISSION_ADMIN';
 const RESOURCE_ORGANIZATION = 'organization';
 const ROLE_ADMIN = 'admin';
 
+function toPromise(keycloakPromise) {
```

Comment:
What is the reason for this?

Reply:
jeg så en random error mens jeg jobbet, så det var en eksperiment. Egentlig kommer det var concept-catalogue-gui, som har typescript. Den ekstra kunnskap å bytte output type ved options er en "afterthought". den internt promise typ er riktig deklarert i TypeScript